subsys/bluetooth/controller/include/ll.h

```diff
@@ -206,3 +206,38 @@ void ll_rx_mem_release(void **node_rx);
 void ll_timeslice_ticker_id_get(uint8_t * const instance_index, uint8_t * const user_id);
 void ll_radio_state_abort(void);
 uint32_t ll_radio_state_is_idle(void);
+
+/* Direction Finding */
+
+/* @brief Function sets CTE transmission parameters for a connection.
+ *
+ * @param[in]handle			Connection handle.
+ * @param[in]cte_types			Bitfield holding information about
+ *					allowed CTE types.
+ * @param[in]switching_patterns_len	Numbe of antenna IDs in switching
+ *					pattern.
+ * @param[in]ant_id			Array od antenna IDs that are antenna
```

Comment:
```suggestion
 * @param[in]ant_id			Array of antenna IDs that are antenna
```
I'm having trouble understanding the rest of this sentence.
"that are antenna switching pattern" what does this mean?

Reply:
yeah that is strange looking - I've changed it to "Array of antenna switch patterns". 
Switch patterns are patterns that are applied to GPIOS setup in Radio to drive antenna switches. 
BT Spec calls these switch patterns.